assets/src/edit-story/components/panels/test/textStyle.js

```diff
@@ -172,6 +172,12 @@ describe('Panels/TextStyle', () => {
     expect(submits[textElement.id]).toStrictEqual({
       fontSize: 70,
       height: 171,
+      lineHeight: NaN,
```

Comment:
Hmm? `NaN` expected?

Reply:
I think it's kinda ok here, since we don't provide all values required to calculate it. We can do `const submits = submit({ fontSize: 70, lineHeight: 1 });` and it will be 1 here. WDYT?

Reply:
Would adding `lineHeight: 1` to the `unlockPaddingTextElement` make a difference?

Reply:
Also does the job, changing.